modules/user/client-react/users/components/UserAddView.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { Link } from 'react-router-dom';
+
+import { translate, TranslateFunction } from '@restapp/i18n-client-react';
+import { PageLayout } from '@restapp/look-client-react';
+
+import UserForm from './UserForm';
+import settings from '../../../../../settings';
+import { User } from '../..';
+
+interface FormValues extends User {
+  password: string;
+  passwordConfirmation: string;
+}
+
+interface UserAddViewProps {
+  t: TranslateFunction;
+  onSubmit: (values: FormValues) => Promise<void>;
+}
+
+const UserAddView = ({ t, onSubmit }: UserAddViewProps) => {
+  const renderMetaData = () => (
```

Comment:
What is the point to duplicate this code snippet in ALL components?

Reply:
Done